### PR TITLE
Add support of cleanDir option

### PR DESCRIPTION
**Pull request by brbrr**
Add support of new cleanDir option: https://github.com/allure-framework/allure-js-commons/pull/19

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -59,10 +59,13 @@ Otherwise choose [one of other ways to generate HTML](https://github.com/allure-
     });
   }
 ```
-Note `done` callback! 
+Note `done` callback!
+
+## Additional allure options
+- `{cleanDir: true}` - will cleanup `allure-results` directory before jasmine starts up
```
Comment (ctapobep) [inline README.md:65]: Could you please update the sample configuration above? I tried this and it didn't work:

```
    jasmine.getEnv().addReporter(new AllureReporter({
      resultsDir: 'allure-results',
      cleanDir: true
    }));
```

File: src/Jasmine2AllureReporter.js [hunk 2]
```diff
@@ -18,11 +18,15 @@ function Jasmine2AllureReporter(userDefinedConfig, allureReporter) {
...
     pluginConfig.basePath = userDefinedConfig.basePath || '.';
-    var outDir = path.resolve(pluginConfig.basePath, pluginConfig.resultsDir);
-    this.allure.setOptions({targetDir: outDir});
+    userDefinedConfig.targetDir = path.resolve(pluginConfig.basePath, pluginConfig.resultsDir);
+    this.allure.setOptions(userDefinedConfig);
   };
   this.configure(userDefinedConfig);
 
+  this.jasmineStarted = function() {
+    this.allure.beforeStart();
```
Comment (ctapobep) [inline src/Jasmine2AllureReporter.js:27]: This fails with `beforeStart is not a function`

**Comment by brbrr** [comment]
@ctapobep sorry, I forgot to mention, that allure-framework/allure-js-commons#19 should be merged before this.

**Comment by just-boris** [comment]
Closing, because the underlying allure-framework/allure-js-commons#19 was not merged

**Pull request closed without merging**